course_catalog/api.py

```diff
@@ -63,6 +64,50 @@ def safe_load_json(json_string, json_file_key):
         return {}
 
 
+def split_ocw_courses_by_run(chunk_size=settings.OCW_ITERATOR_CHUNK_SIZE):
+    """
+    Create a separate course for each OCW run.
+    A reindex will be necessary afterward.
+    """
+    ocw = PlatformType.ocw.value
+    for course in Course.objects.filter(platform="ocw").iterator(chunk_size):
```

Comment:
Filter for published here too?

Reply:
I think the unpublished courses should be processed for consistency (new course_id, 1 run each).